setup.py

```diff
@@ -0,0 +1,36 @@
+import os
+from setuptools import setup, find_packages
+
+here = os.path.abspath(os.path.dirname(__file__))
+
+with open(os.path.join(here, 'README.md')) as f:
```

Comment:
We should convert README in rst because PyPI will not understand the Markdown markup language and only the ReST one.